pkg/minikube/reason/reason.go

```diff
@@ -158,6 +158,8 @@ var (
 	InternalSemverParse = Kind{ID: "MK_SEMVER_PARSE", ExitCode: ExProgramError}
 	// minikube was unable to daemonize the minikube process
 	DaemonizeError = Kind{ID: "MK_DAEMONIZE", ExitCode: ExProgramError}
+	// minikube was unable to add keys to known_hosts
+	AppendKnownError = Kind{ID: "MK_APPEND_KNOWN", ExitCode: ExProgramError}
```

Comment:
```suggestion
	AppendKnownError = Kind{ID: "MK_APPEND_KNOWN_HOSTS", ExitCode: ExProgramError}
```
Let's expand this to prevent any confusion in the future